scripts/run_benchmark.sh

```diff
@@ -22,3 +22,5 @@ cargo run --release --features runtime-benchmarks -- benchmark \
   --heap-pages=4096 \
   --output="${output}" \
   --template=./scripts/frame-weight-template.hbs
+
+cargo fmt
```

Comment:
Why are we adding `cargo fmt` to the benchmark script?

Reply:
it generates the weight.rs file that may or may not cargo fmted. 
Given its a generated file, we can completely avoid lint on that file and many more to come. But I'd rather have us cargo fmt them so that we dont polllute

Reply:
Cool, sounds good